### ISSUE TITLE
Symfony 5 support

### ISSUE THREAD

**Issue by zed-k** [hautelook/AliceBundle#486]
Hi,

Is there any plan to support Symfony 5?
I don't know if a simple version bump in `composer.json` would do the job or if there's deprecations/breaking changes to fix beforehand.

Btw, thanks lot for this bundle to all people involved! 👍 

_Composer output with AliceBundle v2.5.1:_
```
Updating dependencies (including require-dev)
Restricting packages listed in "symfony/symfony" to "5.0.*"
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - hautelook/alice-bundle v2.5.1 requires symfony/framework-bundle ^3.4 || ^4.0 -> satisfiable by symfony/framework-bundle[v4.4.0] but these conflict with your requirements or minimum-stability.
    - hautelook/alice-bundle 2.5.0 requires symfony/framework-bundle ^3.4 || ^4.0 -> satisfiable by symfony/framework-bundle[v4.4.0] but these conflict with your requirements or minimum-stability.
    - hautelook/alice-bundle v2.5.1 requires symfony/framework-bundle ^3.4 || ^4.0 -> satisfiable by symfony/framework-bundle[v4.4.0] but these conflict with your requirements or minimum-stability.
    - Installation request for hautelook/alice-bundle ^2.5 -> satisfiable by hautelook/alice-bundle[2.5.0, v2.5.1].
```

**Comment by olix21** [comment]
It's a little more complicated than that.
The tests are showing deprecation notices that should be fixed first.

There are several notices about the `locateResource` used with deprecated arguments.

**Comment by wblondel** [comment]
That would be great!
nelmio/alice was updated 3 days ago to support Symfony 5

**Comment by tacman** [comment]
Do you think this bundle will need a new version to support Symfony 5?  That might be easier than making it BC.  That is, make this bundle version ^3.0 and only support Symfony 4.4 and Symfony 5, anyone using older versions should stick with Version 2.5.

If it's going to be a new version, that doesn't require BC support, I would LOVE to see a way to escape strings in the fixture files (e.g. headline: "IBM report $200M profit").

**Comment by theofidry** [comment]
@tacman you can check https://github.com/hautelook/AliceBundle/pull/487: I can release a new major version if necessary but as things stands for a minor release will be enough. 

>If it's going to be a new version, that doesn't require BC support, I would LOVE to see a way to escape strings in the fixture files (e.g. headline: "IBM report $200M profit").

I recommend you to check https://github.com/nelmio/alice/issues/998. It is something that can be done without an BC break relatively easily as well :)

**Comment by Gabb1995** [comment]
Any update on when this will be released?

**Comment by theofidry** [comment]
@Gabb1995 there is nothing ready to be released yet... #487 is still unfinished. If you are blocked by it you can consider doing a new PR based on that work to move things forward

**Comment by zed-k** [comment]
#487 merged and released in 2.6.0, thanks a lot to all involved!
